src/core/map/coord.rs

```diff
@@ -65,6 +65,7 @@ impl Coord {
 	/// assert_eq!(Coord::from_linear(-0.5, -1.0), Coord::new_axial(0, -1));
 	/// assert_eq!(Coord::from_linear(-1.5, -1.0), Coord::new_axial(-1, -1));
 	/// ```
+    #[allow(clippy::many_single_char_names)]
```

Comment:
Ugh, why does clippy even worry about that, those are the mathematical variable names of the operations being performed, they *are* single characters and they make sense in this context, not that clippy can really know I guess, but this is useful...